src/tools/add_entity.cc

```diff
@@ -278,15 +209,12 @@ struct add_entity_tool : tool {
                 m[3] = p;
                 break;
             }
-            case PlaceMode::BlockSnapped: {
+            case placement::full_block_snapped: {
                 m = mat_block_face(rc->p, index ^ 1);
```

Comment:
Isn't this the same as the step=1 case?

Reply:
no, because step 1 will not center on block face

Reply:
(Or, if it's not, should it be?)

Reply:
OK, that makes sense.